src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -92,6 +96,7 @@ internal sealed class TaggerProvider : IViewTaggerProvider, ITableDataSource, IS
             this.dte = serviceProvider.GetService<DTE>();
             this.languageRecognizer = languageRecognizer;
             this.logger = logger;
+            this.analysisJobs = new Dictionary<string, CancellationTokenSource>();
```

Comment:
It would be safer to explicitly specify the comparer as being `OrdinalIgnoreCase`.